bundles/binding/org.openhab.binding.mpd/README.md

```diff
@@ -2,11 +2,11 @@
 
 [Music Player Daemon (MPD)](http://www.musicpd.org/) is a flexible, powerful, server-side application for playing music. Through plugins and libraries it can play a variety of sound files while being controlled by its network protocol. 
 
-By help of the openHAB MPD binding, you can e.g. start/stop playing music in specific rooms / on various channels and change volume.
+With the openHAB MPD binding you can e.g. start/stop playing music in specific rooms / on various channels, change volume, and even see which track is playing.
 
 ## Binding Configuration
 
-This binding can be configured in the file `services/mpd.cfg`.
+After installing the MPD version 1 binding by [downloading the .jar file](https://openhab.ci.cloudbees.com/job/PR-openHAB1-Addons/ws/bundles/binding/org.openhab.binding.mpd/target/) into the `addons/` directory, the binding can be configured in the file `services/mpd.cfg`.
```

Comment:
This is about configuration, not installation. Please revert.

Reply:
so where is installation of MPD explained?